src/infra/main/template.yaml

```diff
@@ -357,6 +382,17 @@ Resources:
       LogGroupName: !Ref InterventionsProcessorFunctionLogGroup
       FilterPattern: "{$.message != \"Sensitive info*\"}"
 
+  InterventionsProcessorLambdaErrorMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      FilterPattern: '{ $.message = "Error caught, message will be retried." }'
+      LogGroupName: !Ref InterventionsProcessorFunctionLogGroup
+      MetricTransformations:
+        - MetricValue: 1
+          MetricNamespace: !Ref SystemTagValue
+          MetricName: Interventions_Processor_Lambda_Error
```

Comment:
Can we use UpperCamelCase for Metric names to keep it consistent with other metrics names in the codebase